MonoGame.Framework/Graphics/SurfaceFormat.cs

```diff
@@ -158,31 +158,35 @@ public enum SurfaceFormat
         /// ATC/ATITC compression (Android)
         /// </summary>
         RgbaAtcInterpolatedAlpha = 81,
-
-		/// <summary>
-        /// Etc2 RGB8 (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        Rgb8Etc2 = 90,
-		/// <summary>
-        /// Etc2 SRGB8 (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        Srgb8Etc2 = 91,
-		/// <summary>
-        /// Etc2 RGB8A1 (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        Rgb8A1Etc2 = 92,
-		/// <summary>
-        /// Etc2 SRGB8A1 (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        Srgb8A1Etc2 = 93,
-		/// <summary>
-        /// Etc2 RGBA8 EAC (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        Rgba8Etc2 = 94,
-		/// <summary>
-        /// Etc2 SRGB8A8 EAC (Android/iOS withh OpenglES 3.0)
-		/// </summary>
-        SRgb8A8Etc2 = 95,
+
+		/// <summary>
+        /// Etc2 RGB8 (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        Rgb8Etc2 = 90,
+		/// <summary>
+        /// Etc2 SRGB8 (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        Srgb8Etc2 = 91,
+		/// <summary>
+        /// Etc2 RGB8A1 (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        Rgb8A1Etc2 = 92,
+		/// <summary>
+        /// Etc2 SRGB8A1 (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        Srgb8A1Etc2 = 93,
+		/// <summary>
+        /// Etc2 RGBA8 EAC (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        Rgba8Etc2 = 94,
+		/// <summary>
+        /// Etc2 SRGB8A8 EAC (Android/iOS withh OpenglES 3.0)
+		/// </summary>
+        SRgb8A8Etc2 = 95,
+        /// <summary>
+        /// Adaptive scalable texture compression ; 4x4 matrix using rgba channel interpretation
+        /// </summary>
+        ASTC_4x4_Rgba = 96,

```

Comment:
This is a style/consistency choice. Looking at the other enumerations we historically do not include underscores. Should this be `Rgba4Astc` to fit in with the other tytpes? (or something similar).